go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/webrpc/webrpc
 
 // Webrpc supports the two most recent major versions of Go.
-go 1.22.11
+go 1.22.12
 
-toolchain go1.23.3
+toolchain go1.24.0
```

Comment:
what does "toolchain" do btw..? will this still work with older version of Go, like 1.22 ..?

Reply:
this field can be pretty much ignored, it's written by go command when making changes to `go` version directive

what matters is `go 1.22.3` above -- that's the minimal Go version to be used with this module

https://go.dev/ref/mod#go-mod-file-toolchain